crates/bevy_ecs/src/world/pointer.rs

```diff
@@ -32,18 +35,20 @@ impl<'a, T: core::fmt::Debug> core::fmt::Debug for Mut<'a, T> {
 
 impl<'w, T> Mut<'w, T> {
     /// Returns true if (and only if) this component been added since the start of the frame.
```

Comment:
More outdated doc strings. This is probably a find and replace task of some sort.